augur/export_v2.py

```diff
@@ -1142,6 +1155,17 @@ def run(args):
             parse_node_data_and_metadata(T, node_data_file, metadata_file)
     config = get_config(args)
 
+    # Check additional metadata columns requested exist
+    additional_metadata_columns = []
+    if args.metadata_columns:
+        for col in args.metadata_columns:
+            # Match the column names corrected within parse_node_data_and_metadata
+            corrected_col = update_deprecated_names(col)
+            if corrected_col not in metadata_names:
+                print(f"WARNING: Requested metadata column {col!r} does not exist and will not be exported")
```

Comment:
Suggestion: align the wording of this message with the `--color-by-metadata` counterpart. Currently they are quite different:

```
WARNING: You asked for a color-by for trait 'missing_field', but it has no values on the tree. It has been ignored.
WARNING: Requested metadata column 'missing_field' does not exist and will not be exported
```

I prefer the more concise version. If I were making the change, I would use:

```
WARNING: Requested metadata coloring 'missing_field' does not exist and will not be exported.
WARNING: Requested metadata column 'missing_field' does not exist and will not be exported.
```

Reply:
Updated wording in bcb5e9e456b84697e85dbb816872bb57a0345378